server/src/main/java/tabletop/domain/game/wininformation/SingleWinInformation.java

```diff
@@ -0,0 +1,29 @@
+package tabletop.domain.game.wininformation;
+
+import tabletop.domain.user.User;
+
+import java.util.Set;
+
+public class SingleWinInformation {
+    private Long tournamentId;
+    private User winner;
+
```

Comment:
validation annotations could be added to these fields. Also just user id's could be considered since it's just a DTO.